src/view_control.cpp

```diff
@@ -533,40 +533,43 @@ void View::DrawHairpin(
     FloatingPositioner *leftLink = hairpin->GetCorrespFloatingPositioner(hairpin->GetLeftLink());
     FloatingPositioner *rightLink = hairpin->GetCorrespFloatingPositioner(hairpin->GetRightLink());
 
+    const int unit = m_doc->GetDrawingUnit(staff->m_drawingStaffSize);
+    const hairpinLog_FORM form = hairpin->GetForm();
+    const bool niente = (hairpin->GetNiente() == BOOLEAN_true);
```

Comment:
It usually better to check if an attribute is not default before getting its value. At least that would be necessary if one day we want to have [this](https://github.com/rism-digital/verovio/issues/2545#issuecomment-997668899) type of optimisation

Reply:
I also meant it for `const hairpinLog_FORM form = hairpin->GetForm();`

Reply:
We check for this in line 528.

Reply:
Correct